drltrace_src/drltrace.cpp

```diff
@@ -254,13 +266,48 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
     if (mod != NULL)
         modname = dr_module_preferred_name(mod);
 
+    /* Build the module & function string, then compare to the white/black
+     * list. */
+    char module_name[256];
+    memset(module_name, 0, sizeof(module_name));
+
+    /* Temporary workaround for VC2013, which doesn't have snprintf().
+     * apparently, this was added in later releases... */
+#ifdef WINDOWS
+#define snprintf _snprintf
+#endif
+    size_t module_name_len = (size_t)snprintf(module_name, sizeof(module_name) - 1, "%s%s%s", modname == NULL ? "" : modname, modname == NULL ? "" : "!", name);
```

Comment:
drltrace used to follow DynamoRIO code style. We usually try to avoid lines longer than 90 chars.